workshops/meme-generator/README.md

```diff
@@ -0,0 +1,379 @@
+You can take a look at how different input types work [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
+
+### 2) CSS
+
+We'll add some basic styling to our website:
+
+```CSS
+h1 {
+    font-family: Impact, 'Arial Narrow Bold', sans-serif;
+    font-size: 30px;
+}
+
+body {
+  margin: 10px;
+  background-color: lightblue;
+}
+
+#meme-canvas {
+    width: 300px;
+}
+```
+
+So in this CSS code, we first change our heading's font and font size, then we apply a background color to our body, and at last we select the meme canvas by it's ID and set its width to 300 pixels.
+
+That's it! This is all the CSS we'll be doing in the workshop.
+
+### 3) JavaScript
+
+Now, in our `scripts.js` file:
+
+* Make a `function` called `generateMeme`, which will take: `image`, `topText`, `bottomText`, `topTextSize`, `bottomTextSize` variables as the function arguments.
+  
+  Here's how our `generateMeme` function will look like:
+  
+  ```javascript
+  function generateMeme(img, topText, bottomText, topTextSize, bottomTextSize) {
+  const canvas = document.getElementById('meme-canvas');
+  const ctx = canvas.getContext('2d');
+
+  // Size canvas to image
+  canvas.width = img.width;
+  canvas.height = img.height;
+
+  // Clear canvas
+  ctx.clearRect(0, 0, canvas.width, canvas.height);
+  // Draw main image
+  ctx.drawImage(img, 0, 0);
+
+  // Text style: white with black borders
+  ctx.fillStyle = 'white';
+  ctx.strokeStyle = 'black';
+  ctx.textAlign = 'center';
+
+  // Top text font size
+  let fontSize = canvas.width * topTextSize;
+  ctx.font = `${fontSize}px Impact`;
+  ctx.lineWidth = fontSize / 20;
+
+  // Draw top text
+  ctx.textBaseline = 'top';
+  topText.split('\n').forEach((t, i) => {
+    ctx.fillText(t, canvas.width / 2, i * fontSize, canvas.width);
+    ctx.strokeText(t, canvas.width / 2, i * fontSize, canvas.width);
+  });
+
+  // Bottom text font size
+  fontSize = canvas.width * bottomTextSize;
+  ctx.font = `${fontSize}px Impact`;
+  ctx.lineWidth = fontSize / 20;
+
+  // Draw bottom text
+  ctx.textBaseline = 'bottom';
+  bottomText.split('\n').reverse().forEach((t, i) => { // .reverse() because it's drawing the bottom text from the bottom up
+    ctx.fillText(t, canvas.width / 2, canvas.height - i * fontSize, canvas.width);
+    ctx.strokeText(t, canvas.width / 2, canvas.height - i * fontSize, canvas.width);
+    }
+    ```
+
+  If you wanna learn more about the cool things `Canvas` can do, check out [this](https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API/Tutorial/Basic_usage) link.
+
+* Now, after our `generateMeme` function, add a [Window: DOMContentLoaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event) event, which will be executed after the content of the web page is loaded.
+
+    ```javascript
+    window.addEventListener('DOMContentLoaded', () => {
+        //Code here
+    });
+    ```
+
+Inside the event, add the following code:
```

Comment:
Wait this is weird because all the text before this was inside a list, but suddenly the list ends. This looks weird